tests/unit/test_releases.py

```
@@ -1,123 +1,304 @@
-    def test_list_releases(self, client: PlaneClient, workspace_slug: str) -> None:
-        """Test listing releases."""
-        releases = client.releases.list(workspace_slug)
-        assert isinstance(releases, list)
+    @pytest.fixture
+    def release(self, client: PlaneClient, workspace_slug: str):
+        rel = client.releases.create(workspace_slug, CreateRelease(name=f"release-{_uid()}"))
+        yield rel
+        try:
+            client.releases.delete(workspace_slug, rel.id)
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)